scout/server/blueprints/cases/templates/cases/clinvar_submissions.html

```diff
@@ -144,9 +144,9 @@ <h4>Variant data:</h4>
                       <div class="vardata">
                         <div id="vardiv{{subm_variant._id}}" class="panel-body" style="display:none;">
                           {% if subm_variant.category == 'sv' %}
-                          <a href="{{ url_for('variants.sv_variant', institute_id=institute_id, case_name=subm_variant.case_id, variant_id=subm_variant.local_id) }}" target="_blank"><h4>{{subm_variant.chromosome}}_{{subm_variant.breakpoint1}}_{{subm_variant.var_type}}</h4></a>
+                          <a href="{{ url_for('variants.sv_variant', institute_id=institute_id, case_name=subm_obj.cases[subm_variant.case_id], variant_id=subm_variant.local_id) }}" target="_blank"><h4>{{subm_variant.chromosome}}_{{subm_variant.breakpoint1}}_{{subm_variant.var_type}}</h4></a>
```

Comment:
Looks ok, but to me those hash-ish IDs aren't that informative. How about the display version with chromosome_coord_ref_alt - and truncated if from an SV or larger MNV?

Reply:
Good suggestion! It's more informative for the users. I'll change it!